Dockerfile

```diff
@@ -22,10 +22,4 @@ SHELL ["/bin/bash", "-c"]
 # Activate the conda environment by default when starting a shell
 # This will ensure that the conda environment's python is used
 RUN echo "source activate molkart_local" > ~/.bashrc
-ENV BASH_ENV ~/.bashrc
-
-# Set the working directory
-WORKDIR /local
-
-# Copy the current directory contents into the container at /local
-COPY . .
```

Comment:
If this is removed, scripts from the repo aren't executible, this might have to be readded later. 
Is there another reason why you remove this line?

Reply:
Ahh I forgot about the scripts. I don't really like the idea of having duplicate of the scripts in here, I just removed it. But yeah if we want to keep the scripts we would have to copy the folder. 

My thinking behind removing the folder was, that we don't really use anything from the repo in the container (again, I forgot about the scripts). To me the container is just a preset box with all the packages etc installed for our modules.

Reply:
With duplicates, I meant duplicates relative to the scripts within the pipeline. I just feel like this creates another copy of a script and then if we don't update them (like we didn't do now for maskfilter (my bad ;)), then they are out of date anyways.

Reply:
For the moment, I have added this back to the container. There are still a few improvements to be made for the usage of the container in gitpod, but I would like to merge this PR for the moment to use anndata in molkart!